examples/kube-state-metrics.yaml

```diff
@@ -259,6 +259,36 @@ rules:
   - list
   - watch
 ---
+apiVersion: autoscaling/v2beta2
```

Comment:
Why not `autoscaling/v2`?

Reply:
Only available on 1.23+ clusters - https://kubernetes.io/blog/2021/12/07/kubernetes-1-23-release-announcement/#horizontalpodautoscaler-v2-graduates-to-ga